day-01/src/app.js

```diff
@@ -1,39 +1,34 @@
 import React from 'react';
```

Comment:
import Rect, { Component } from 'react';
then you can extend the Component itself